cmake/Utilities.cmake

```diff
@@ -58,7 +58,7 @@ endfunction()
 function(report_size target)
   add_custom_command(
     TARGET ${target} POST_BUILD
-    COMMAND echo "" # visually separate the output
+    COMMAND ${CMAKE_COMMAND} -E echo "" # visually separate the output
```

Comment:
I almost think this extra echo here is not useful. The output looks already separated to me.

Reply:
Looks like this without it. I don't mind deleting this line. ![image](https://user-images.githubusercontent.com/8218499/215285585-e5f0f7b8-1422-4c58-8f7e-ae2acf15be74.png)